wallets/metamask/src/config.ts

```diff
@@ -0,0 +1,11 @@
+import dotenv from 'dotenv'
+import findConfig from 'find-config'
+
+export const loadEnv = () => {
+  const envFiles = ['.env', '.env.e2e', '.env.local', '.env.dev']
```

Comment:
what is the use case for '.env.local' vs '.env.dev'?

Reply:
same as .env, mostly used during local development with free keys vs paid prod keys